zhaquirks/tuya/doorsensor.py

```diff
@@ -0,0 +1,63 @@
+"""Doorsensors"""
+
+import zigpy.profiles.zha as zha_p
```

Comment:
Import this as just `zha` to be more consistent with other quirks.

Reply:
Done, Thanks